package.json

```diff
@@ -47,7 +47,7 @@
     "prettier": "2.5.1",
     "rollup": "^2.66.1",
     "rollup-plugin-bundle-imports": "^1.5.1",
-    "rollup-plugin-chrome-extension": "^4.0.1-12",
+    "rollup-plugin-chrome-extension": "4.0.1-11",
```

Comment:
This rollback is necessary because the assets are broken after the introduction of `4.0.1-12`